Source/Redemption/Private/PauseMenuWidget.cpp

```diff
@@ -26,10 +26,11 @@ void UPauseMenuWidget::Exit()
 	//Opens the "MainMenu" Map
 	UGameplayStatics::OpenLevel(GetWorld(), "MainMenu");
 
+	/*
 	APlayerController* Controller = UGameplayStatics::GetPlayerController(GetWorld(), 0);
 	Controller->SetInputMode(FInputModeUIOnly());
 	Controller->bShowMouseCursor = false;
-
+	*/
```

Comment:
Can this code be removed now?

Reply:
yes, I will create a branch later to delete all commented code and take out UE_LOGs